onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -1411,14 +1409,14 @@ common::Status TensorrtExecutionProvider::Compile(const std::vector<Node*>& fuse
                 if (tensor_shape_value < shape_range[j].first) {
                   shape_range[j].first = tensor_shape_value;
                   shapes_min[j] = tensor_shape_value;
-                  dimension_update[input_name] = true;
+                  engine_update = true;
```

Comment:
dimension_update is to keep tracking dynamic shape tensors with shape updates. It will be used later in setInputShapeBinding()

Reply:
Sorry, I changed that but forgot to commit.